gpkit/nomials/nomial_math.py

```diff
@@ -118,9 +118,9 @@ def __init__(self, exps=None, cs=1, require_positive=True, simplify=True,
             self.exp = self.exps[0]
             self.c = self.cs[0]
 
-    def to(self, arg):
+    def convert_to(self, arg):
         "Convert to in the pint units sense"
-        return Signomial(self.exps, self.cs.to(arg).tolist())
+        self.cs = self.cs.to(arg)
```

Comment:
shall we leave the `to` method for users, and add `convert_to` as the in place version as well?